### DISCUSSION TITLE
Suggestions on how to mock data for NoSQL database

### DISCUSSION THREAD

**Discussion by rowlandekemezie** [mswjs/data#116] [Q&A]
My application is data-driven but developed with Non-relational Database.

I think using the current package doesn't mirror the application code. So, I'm wondering if there's any solution or suggestion on how to approach the mocking.

**Comment by kettanaito** [comment]
Hey, @rowlandekemezie. Could you please share some more info on your NoSQL data structure?

The usage of this package resembles describing a document store, and I can imagine it being useful for your use-case as well.
